build.gradle

```diff
@@ -66,7 +66,7 @@ final testNGVersion = '6.11'
 
 final baseJarName = 'gatk'
 final secondaryBaseJarName = 'hellbender'
-
+final docBuildDir = 'build/docs'
```

Comment:
this should be using $buildDir.

Reply:
Had to change some single quote to doubles to enable interpolation. Unrelated note: is there any downside to standardizing on double quotes everywhere ?

Reply:
Done.